src/components/minimap-toolbar/minimap-toolbar.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../styles/variables';
+@import '../../styles/extends';
+
+.pipeline-minimap-toolbar {
+  display: block;
+  padding: 0;
+  list-style: none;
+
+  .pipeline-icon-toolbar__button {
```

Comment:
This nesting is unnecessary - that's why we use BEM

Reply:
We should never be nesting classes if we don't have to, because it unnecessarily increases CSS specificity. This is the main motivation behind BEM: To allow class combinations without increasing specificity. The above would output:
```css
.pipeline-minimap-toolbar { ... }
.pipeline-minimap-toolbar .pipeline-icon-toolbar__button { ... }
```
which is more specific than the following, without giving any benefit:
```css
.pipeline-minimap-toolbar { ... }
.pipeline-icon-toolbar__button { ... }
```

The theme classes (i.e. `.kui-theme--light & {}`) are also nested, which is unfortunate but necessary, given the way Kedro-UI applies theming. I'd like to fix this with CSS custom properties in future, but for now it'll have to stay that way.

Reply:
The increased specificity was intended in this case because that descendent button is inside another component whose style is not owned by the host component, but instead of this I was able to remove the nesting by using a unique classname instead.